src/styles/antd-overrides/input.scss

```diff
@@ -292,3 +296,7 @@ input:-webkit-autofill-selected {
   margin-bottom: 0;
   width: 100%;
 }
+
+.ant-form-item-optional {
```

Comment:
@johnnyd-eth ah, I added this in https://github.com/jbx-protocol/juice-interface/pull/1234/files :)

Also we should def try to avoid `!important`. Can 99% of the time get around it with greater [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).